.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+indent_style = space
```

Comment:
You've added this but then all the spacing has been changed to tabs? Eh?

Reply:
I _believe_ it's the other way around

Reply:
![image](https://user-images.githubusercontent.com/3260989/70006876-69208900-1523-11ea-97ba-990d54df3927.png)

Notice the whitespace markers 😛

Reply:
smh